frontend/test/__support__/ui.tsx

```diff
@@ -8,69 +7,75 @@ import { Provider } from "react-redux";
 import { ThemeProvider } from "@emotion/react";
 import { DragDropContextProvider } from "react-dnd";
 import HTML5Backend from "react-dnd-html5-backend";
+
 import { state as sampleDatabaseReduxState } from "__support__/sample_database_fixture";
+
+import type { User } from "metabase-types/api";
+import type { State } from "metabase-types/store";
+
+import { createMockUser } from "metabase-types/api/mocks";
 import {
   createMockSettingsState,
   createMockEmbedState,
 } from "metabase-types/store/mocks";
+
 import { getStore } from "./entities-store";
 
-function getUser(user = {}) {
-  return {
-    id: 1,
-    first_name: "Bobby",
-    last_name: "Tables",
-    email: "bobby@metabase.test",
-    is_superuser: true,
-    ...user,
-  };
+type DeepPartial<T> = T extends object
```

Comment:
We can't fill in default values with this approach. I think it should be just `Partial`, and calls sites should be like this:
```
const initialState = createMockState({
  qb: createMockQbState({
     visibleTimelineIds: [1, 2, 3]
  }),
});
```

WDYT?